contracts/bridge-v1.yml

```diff
@@ -2053,4 +2111,46 @@ components:
         careContextReference:
           type: string
           example: NCC1701
-          description: care context reference number.
\ No newline at end of file
+          description: care context reference number.
+    HipInitiatedLinkResult:
+      type: object
+      properties:
+        requestId:
+          type: string
+          format: uuid
+          description: a nonce, unique for each HTTP request
+          example: 5f7a535d-a3fd-416b-b069-c97d021fbacd
+        timestamp:
+          $ref: '#/components/schemas/TimestampSchema'
+        meta:
```

Comment:
This needs to go into the header, instead of body, right?

Reply:
meta right?